### PR TITLE
Add discord bot

### PR DESCRIPTION
**Pull request by IsaLjungdahl**
Simple discord-bot with eventlisteners, it can respond to !ping. Atm live in our discord-server.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,6 @@
-# Nothing here yet
\ No newline at end of file
+# SonarQuBot
+
+To make this work with your discord-bot you need to add your bot-token to an app.properties file inside the root-map.
+<br>
```
Comment (DoNotWorr) [inline README.md:4]: Mapp = folder

File: src/main/java/org/five/sonarqubot/SonarqubotApplication.java [hunk 2]
```diff
@@ -10,4 +10,6 @@ public static void main(String[] args) {
         SpringApplication.run(SonarqubotApplication.class, args);
     }
 
+
```
Comment (DoNotWorr) [inline src/main/java/org/five/sonarqubot/SonarqubotApplication.java]: Unnecessary empty lines (might cause merge conflicts)

**Review by DoNotWorr — APPROVED** [review]
Only some spelling mistakes

File: README.md (hunk 1 above)
Comment (IsaLjungdahl) [reply README.md:4]: hahah!

**Review by Abdih10 — APPROVED** [review]

**Pull request merged**